xml/schema/Core/Job.xml

```diff
@@ -61,6 +61,13 @@
     <add>4.1</add>
   </field>
   <field>
+    <name>scheduled_run_date</name>
+    <type>timestamp</type>
+    <default>NULL</default>
```

Comment:
@eileenmcnaughton Both of the other `timestamp NULL DEFAULT NULL` fields I referenced both have `<default>NULL</default>` in their schema definition, but they both also have `<required>false</required>` which my definition does not. Do you think if I add that to Job.xml that it would pass the test?